.github/workflows/main.yml

```diff
@@ -95,6 +95,8 @@ jobs:
         run: |
           # build docker and run test
           just docker-test
+          # test the prod image builds, as its slightly different, and has caught us out before.
```

Comment:
I'm confused. Reading through the docker-compose file, should the prod image get built since the dev one is built from it, or is there some caching happening? 

If this is required, would it make sense to put this step before the test one?

Reply:
The prod build target does not get built when we build `interactive-dev`, as `interactive-dev` is build `FROM interactive-base`. 

The `interative-prod` build target is mostly just adding metadata, but it also has the collectstatic build step, which is specifically what was not tested it PR test workflow, and failed in the post-merge deploy workflow.

Argubly, we could instead have `just docker-test` run both the tests *and* build the prod image, rather than specifically adding that step to the GH action? Then we'd get feedback in dev if we've broken the prod image build somehow

Reply:
Yes, thank you. That makes sense. Thinking about it some more, I think the change you've made here makes more sense. Having `just docker-test` build an unrelated docker image would be a weird side effect of the command, unless it was actually inside a test.